README.md

```diff
@@ -314,15 +334,15 @@ The format is `python_tag-platform_tag`. The tags are similar but not identical
 
 Python tags look like `cp27` `cp34` `cp35` `cp36` `cp37`
 
-Platform tags look like `macosx_10_6_intel` `manylinux_x86_64` `manylinux_i686` `win32` `win_amd64`
```

Comment:
Could we do one of those identifier migrations like how @YannickJadoul did the `manylinux1`->`manylinux` warnings?

Reply:
added

Reply:
added